src/main/java/org/folio/circulation/domain/validation/CommonFailures.java

```diff
@@ -18,8 +18,9 @@ private CommonFailures() { }
 
   public static Supplier<HttpFailure> noItemFoundForBarcodeFailure(String itemBarcode) {
     return () -> singleValidationError(
-      format("No item with barcode %s exists", itemBarcode),
-      "itemBarcode", itemBarcode);
+        "ui-circulation.mod-circulation.noItemFoundForBarcode",
```

Comment:
I love the idea of namespacing here, but I'm not sold on this implementation. If anything, I would expect the namespace to be tied to the interface this module implements rather than the module name itself (as seen in `mod-circulation` substring) because other modules could implement this interface, or the name of a UI module which potentially consumes this interface (as seen in the `ui-circulation` substring) because a server-side module is necessarily unaware of a UI module and, again, other UI modules could consume this interface. 

In the UI, we recently implemented functions to handle namespacing for us automatically. A function like `getNamespace()` (with optional params when you want to override defaults) can also prevent typos in strings because the compiler will whine about `getNamespice()` whereas a string like `noItemFoumdForBarcode` may be harder to catch.

Additionally, I would expect a constant value like this to use `SNAKE_CASE`, as do [other](https://github.com/folio-org/mod-inventory/blob/644ee9195acdf8fd6e64601c87bb98a909148cf1/src/main/java/org/folio/inventory/common/MessagingContext.java) [FOLIO](https://github.com/folio-org/okapi/blob/9a2d5216d820c684a30c3c6fc0b21ff024c37e3a/okapi-core/src/main/java/org/folio/okapi/ConfNames.java) [repos](https://github.com/folio-org/edge-common/blob/50560250bb66015e1b992448921f41bc56ccee84/src/main/java/org/folio/edge/core/Constants.java).

Reply:
Whoops, I should have scrolled down where I could have seen that @marcjohnson-kint has already described many of these same concerns.

Reply:
> I've created https://issues.folio.org/browse/STRIPES-750 "DECISION: Which module should host back-end translation files?" to address the issues you've described.

Ok, thank you.

(I don't really understand why this is considered a front end only decision made by the Stripes Force team)